Farm.java

```diff
@@ -0,0 +1,24 @@
+package ua.lviv.iot.firstLab;
+
+public record Farm(String Location, int NumberOfAnimals, int PowerOfFans, int NumberOfEmployee, String NameOfOwner, boolean Export) {
```

Comment:
Please pay attention to namings. For variables and methods it should be mixed case with a lowercase first letter (e.g. String likeThis)